backend/src/config.ts

```diff
@@ -13,6 +13,11 @@ export interface IConfig {
   sourceRepo: string;
   hldRepo: string;
 
+  // gitlab
```

Comment:
Just a styling thing: Since all these are set via `getConfig`, might want to use optional properties instead (i.e. `hldRepo?: string`) then you can don't need all the `||` shortcircuting in `getConfig`